### DISCUSSION TITLE
Can I add a prefix to the generated class ?

### DISCUSSION THREAD

**Discussion by iam-frankqiu** [tailwindlabs/tailwindcss#3055] [Ideas]
When I am developing a Chrome extension. I need to identify the class between the extension and the original website.

**Comment by simonswiss** [comment]
Hey!

Not sure if I understand the question properly, but there is a `prefix` option in the Tailwind config where you can define a `prefix` that will be applied to all generated classes:

https://tailwindcss.com/docs/configuration#prefix

Hope it helps 👍
